.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": [
+    "react-app",
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": ["react", "prettier", "import"],
+  "rules": {
+    "prettier/prettier": [2, { "singleQuote": true }],
+    "import/no-extraneous-dependencies": 0,
```

Comment:
Can you remove the rules we don't need? I.e. remove all and re-add them if an error occurs. Most of them are only necessary for the project I linked to in #18. `prettier/prettier` should at least stay, but I believe the others aren't necessary.

Reply:
👍 Changed it, did you mean it like I did it now?